client/src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+class Header extends Component {
```

Comment:
This could be a [functional component](https://reactjs.org/docs/components-and-props.html).